python_requests/example.py

```diff
@@ -0,0 +1,81 @@
+"""
+Usage
+
+python example.py <hotel_id> <partner_id> <api_key>
+"""
+import argparse
+import base64
+
+import requests
+
+
+def get_auth_token(partner_name, api_key):
+    """Returns the API authorization token. Use this for subsequent calls.
+ 
+    When the token expires, you can get another one using logic similar to what this function
+    illustrates
+
+    :param str partner_name: the partner's name for use in the API
+    :param str api_key: the partner's API key, for use in the API
+
+    :rtype: str
+    :return: A JWT token
+    """
+    authorization_raw_key = '{}:{}'.format(partner_name, api_key)
+
+    auth_response = requests.post(
+        'https://analytics.trustyou.com/connect/api/v1.0/auth',
+        headers={
+            'Authorization': 'Basic {}'
+            .format(base64.b64encode(authorization_raw_key.encode('ascii')).decode('ascii')),
```

Comment:
Can you offload this base64 expression to another variable defined under the `authorization_raw_key` above? The naming will make it easier for people to imagine the purpose of this expression even without knowing any Python.

For example: 
```
base64_encoded_key = base64.b64encode(authorization_raw_key.encode('ascii')).decode('ascii')
```

And then the `headers` dictionary would like the following example:

```
headers = {
    'Authorization': 'Basic {}'.format(base64_encoded_key),
    ...
}
```